jenkins-slaves/jenkins-slave-argocd/Jenkinsfile.test

```diff
@@ -0,0 +1,30 @@
+pipeline {
+    agent {
+        kubernetes {
+            label 'jenkins-slave-argocd'
+            cloud 'openshift'
+            serviceAccount 'jenkins'
+            containerTemplate {
+                name 'jnlp'
+                image "image-registry.openshift-image-registry.svc:5000/openshift/jenkins-slave-argocd"
+                alwaysPullImage true
+                workingDir '/tmp'
+                args '${computer.jnlpmac} ${computer.name}'
+                command ''
+                ttyEnabled false
+            }
+        }
+    }
```

Comment:
I believe you can shorten this to
```
pipeline {
    agent {
        label 'jenkins-slave-argocd'
    }
}
```
What you have now works, just curious if there's a particular reason for it?